xapian-core/api/omenquireinternal.h

```diff
@@ -313,6 +313,21 @@ class MSet::Internal : public Xapian::Internal::intrusive_base {
 	/** Fetch items specified into the document cache.
 	 */
 	void fetch_items(Xapian::doccount first, Xapian::doccount last) const;
+
+	/** Updates the weight corresponding to the document indexed at position Xapian::docount i with double wt_.
```

Comment:
You don't need the types here, just the variable names would make it clear enough.